docs/_posts/2017-10-19-diversity-open-source.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "Diversity in Open Source, and Jekyll's role in it"
+date: 2017-10-19 21:33:00 +0200
+author: pup
+categories: [community]
+---
+
+Open Source has a problem with diversity. GitHub recently conducted a [survey](http://opensourcesurvey.org/2017) that revealed that 95% of the respondents were male, making the demographic even worse than in tech overall. Every other week, I seem to hear of another case of a maintainer engaging in targeted harassment against minorities. People somehow deem it completely okay to let these things slide, though.
+
+So yeah, there's a bunch of big problems plaguing open source. Fortunately, there's a couple of things we can do to make it easier and more comfortable for people, especially minorities, that have never before contributed to any open source project before, to contribute to our projects.
```

Comment:
I'd strike the `So yeah` sentence and dive right into `Fortunately`. I also wonder if you can strike "especially minorities" – you are saying "people that have never before contributed" and I think the goal there is fairly direct. Ah, and you have two `before`'s in that last sentence.